.github/pull_request_template.md

```diff
@@ -0,0 +1,43 @@
+Thank you for contributing to this repository!
+
+You can help us review your changes by answering these questions.
+They are all optional, but the more information you provide the easier it will be for us to review your changes.
+Everyone here is a volunteer, so please help us out if you can.
+
+If you want to discuss anything Jupyter related, or to meet other users and developers, please say hello on https://discourse.jupyter.org/ .
+
+
+### What does this PR do?
+<!--
+Please summarise this PR.
+Please also edit the PR title so that it contains enough context to go into a changelog.
+Use "Fixes #<NUM>" if this fixes an existing issue.
+It may help to list each change with an explanation of why it's needed- remember that what seems obvious to you may not be obvious to a reviewer.
+-->
+
+### Is this PR related to another issue, or is it part of a larger body of work?
+<!--
+Please feel free to provide as much context or links to external sites as you want!
+-->
+
+### Does this PR introduce a breaking change?
+<!--
+If so what changes might users need to make in their applications due to this PR?
+-->
+
+### How can this PR be tested?
+<!--
+If this is not fully covered by the automated tests please help us by showing us the steps needed to verify the changes work.
+-->
```

Comment:
```suggestion
<!--
If this is not fully covered by the automated tests please help us by describing the tests that you ran to verify your changes. Make sure to provide as much detail so that we can reproduce your tests.
-->
```